sdks/java/core/src/main/java/org/apache/beam/sdk/util/UserCodeException.java

```diff
@@ -29,11 +30,18 @@
 public class UserCodeException extends RuntimeException {
 
   public static UserCodeException wrap(Throwable t) {
+    return wrap("", t);
+  }
+
+  public static UserCodeException wrap(String step, Throwable t) {
```

Comment:
`String step` -> `@Nullable String stepName`, and check only vs `null` instead of `isNullOrEmpty`

Reply:
Done